worker/serve.go

```diff
@@ -92,9 +92,6 @@ func serveContent(rw http.ResponseWriter, req *http.Request, obj api.Object, dow
 	rw.Header().Set("ETag", api.FormatETag(buildETag(req, obj.ETag)))
 	rw.Header().Set("Content-Type", contentType)
 
-	// override the range request header to avoid seeks in http.ServeContent
-	req.Header.Set("Range", fmt.Sprintf("bytes=%d-%d", offset, offset+length-1))
```

Comment:
In the `http` package in fs.go:315  there's `case len(ranges) > 1:` that will start seeking multipart byte ranges in the content. We can't have that so we have to override the range header I think.